src/Promitor.Agents.Scraper/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can you remove this file please?

Reply:
Done ! God Damn Visual Studio sometimes !

Reply:
Thanks and no worries!